package.json

```diff
@@ -25,9 +25,11 @@
     "dompurify": "^3.1.6",
     "easytimer.js": "^4.6.0",
     "element-plus": "^2.7.6",
+    "emojilib": "^3.0.12",
     "eslint": "^8.57.0",
     "eslint-import-resolver-custom-alias": "^1.3.2",
     "eslint-plugin-vue": "^9.27.0",
+    "fuse.js": "^7.0.0",
```

Comment:
I'm pretty impressed with this library, both in terms of its performance and also the quality of its matching algorithm.